### PR TITLE
PR for Fixing the alignment of avatar and the paragraph.

### PR DESCRIPTION
**Pull request by umarshaikh97**
Hi, I noticed a small issue that the avatar and the paragraph was not center aligned, so I fixed the alignment of avatar and the paragraph.

Before Fix:
abc

After Fix:
ok

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -59,11 +59,14 @@
...
 .paragraph {
+  display: flex;
+  align-items: center;
   font-size: 2.6vh;
 }
 
 .paragraph a{
-    color: rgba(0,0,0,.45);
+  color: rgba(0,0,0,.45);
+  margin-right: 10px;
```
Comment (NomanGul) [inline src/App.css]: @umershaikh97 10px margin not looking good in mobile view... change margin-right to `margin-right: 1vh;`

**Review by NomanGul — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 1 above)
Comment (umershaikh97) [reply src/App.css]: Sure

Comment (umarshaikh97) [reply src/App.css]: @NomanGul Done!

**Comment by NomanGul** [comment]
@all-contributors please add @umershaikh97 for code

**Pull request merged**